articles/LIV_Plot_Sarthak/info.json

```diff
@@ -5,24 +5,22 @@
+    "tags": ["visualization"],
+    "platforms": ["tardis"],
+    "short_description": "This article highlights the development of the LIV (Last Interaction Velocity) Plot during this summer's GSoC program—a new visualization tool that plots the last photon packet interactions by velocity for each element in supernova ejecta, offering deeper insights into supernova spectra.",
     "cover_image_height": "520px",
     "cover_image_width": "330px",
     "cover_image": "research_news/articles/LIV_Plot_Sarthak/media/images/LIV_ply.png",
     "content": {
-        "1_para": "We can create a highly informative and visually appealing Last Interaction Velocity plot using Matplotlib.",
-        "2_img": "research_news/articles/LIV_Plot_Sarthak/media/images/LIV_mpl.png",
-        "3_para": "If you're using the Last Interaction Velocity plot for exploration, consider creating an interactive version with Plotly. This allows you to zoom, pan, and inspect data values by hovering, resizing the scale, and more conveniently.",
-        "4_img": "research_news/articles/LIV_Plot_Sarthak/media/images/plotly-gif.gif",
-        "5_para": "The detailed report for the GSoC project is available <a href='https://sites.google.com/view/sarthak-gsoc2024/home' target='blank'>here</a>."
+        "1_para": "The TARDIS SDEC plot visualizes which atoms photon packets interact with before they escape the ejecta and contribute to the output spectrum's absorption and emission features. However, the SDEC plot does not visualize where these interactions occur within the ejecta. This information can be beneficial to astronomers who want to understand which regions of the ejecta are responsible for important features in the output spectrum.",
+        "2_para": "This project develops the LIV (Last Interaction Velocity) Plot that plots the number of photon packet interactions as a velocity function for each element present in the ejecta. One such example is shown in the plot below.",
+        "3_img": "research_news/articles/LIV_Plot_Sarthak/media/images/LIV_mpl.png",
+        "4_para": "We can create a highly informative and visually appealing Last Interaction Velocity plot using Matplotlib, If you're using the Last Interaction Velocity plot for exploration, consider creating an interactive version with Plotly. This allows you to zoom, pan, and inspect data values by hovering, resizing the scale, and more conveniently.",
+        "5_vid": "research_news/articles/LIV_Plot_Sarthak/media/images/plotly-gif.gif",
+        "6_para": "The plots have various options providing the users with more control over how your last interaction velocity looks, like, 'species_list', 'nelements', 'packet_wvl_range', 'num_bins', 'log_scale' and many other options.",
+        "7_para": "The detailed report for the GSoC project is available at https://sites.google.com/view/sarthak-gsoc2024/home"
```

Comment:
Can you replace the link here with this:
<a href="https://sites.google.com/view/sarthak-gsoc2024/home" target="_blank">https://sites.google.com/view/sarthak-gsoc2024/home</a>